subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/impl/AbsolutePathFileCollectionFingerprinterTest.groovy

```diff
@@ -63,21 +63,21 @@ class AbsolutePathFileCollectionFingerprinterTest extends Specification {
         def fingerprint = fingerprinter.fingerprint(files(file, dir, noExist))
 
         then:
-        fingerprint.fingerprints.keySet().collect { new File(it) } == [file, dir, dir2, file2, noExist]
+        fingerprint.fingerprints.keySet().collect { new File(it) } == [file, dir, dir2, file2]
     }
 
     def "retains order of elements in the snapshot"() {
         given:
         TestFile file = tmpDir.createFile('file1')
-        TestFile file2 = tmpDir.file('file2')
-        TestFile file3 = tmpDir.file('file3')
+        TestFile noExist = tmpDir.file('file2')
```

Comment:
How about `missing`?

Reply:
In other tests we also keep the variable name and the file name the same where it makes sense, I think we should follow that here too.

Reply:
I renamed the variable and the file names.